pallets/collator-selection/src/mock.rs

```diff
@@ -124,6 +124,8 @@ impl pallet_aura::Config for Test {
 	type MaxAuthorities = ConstU32<100_000>;
 	type DisabledValidators = ();
 	type AllowMultipleBlocksPerSlot = ConstBool<false>;
+	#[cfg(feature = "experimental")]
+	type SlotDuration = pallet_aura::MinimumPeriodTimesTwo<Self>;
```

Comment:
Is there a difference between this and `ConstU64<SLOT_DURATION>`? @rphmeier

Reply:
Not in practice.